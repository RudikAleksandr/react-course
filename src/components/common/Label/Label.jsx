import React from 'react';
import { FormLabel } from 'react-bootstrap';
import cn from './Label.module.css';

const Label = ({ className = '', text = '', ...props }) => (
  <FormLabel
    className={`${cn.label} ${className}`}
    {...props}
  >
    {text}
  </FormLabel>
);

export default Label;
