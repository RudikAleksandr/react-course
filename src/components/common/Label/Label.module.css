.label {
  color: #f65261;
  text-transform: uppercase;
  font-weight: bold;
}