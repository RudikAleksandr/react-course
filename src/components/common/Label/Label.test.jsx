import React from 'react';
import { render } from '@testing-library/react';
import Label from './Label';

describe('Label component', () => {
  test('render Label snapshot', () => {
    const { asFragment } = render(<Label />);

    expect(asFragment(<Label />)).toMatchSnapshot();
  });
});
