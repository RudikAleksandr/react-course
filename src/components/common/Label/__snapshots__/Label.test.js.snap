// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component render Label snapshot 1`] = `
<DocumentFragment>
  <label
    class="label  form-label"
  />
</DocumentFragment>
`;
